src/components/ResultsGrid.tsx

```diff
@@ -0,0 +1,184 @@
+import { Cell, Column, Row, Table, TableHeader } from './Table';
+
+const STATION_NAME = 'STATION_NAME';
+type Result = Record<string, string | number | null>;
+async function getData(where: string, currentUser: User): Promise<Iterable<Result>> {
+  if (where === '') {
+    return [];
+  }
+
+  const query = `
+  SELECT DISTINCT
+      se.${config.fieldNames.EVENT_ID},
+      ${config.fieldNames.EVENT_DATE},
+      ${config.fieldNames.OBSERVERS},
+      l.${config.fieldNames.WaterName} as ${config.fieldNames.WaterName}_Lake,
+      l.${config.fieldNames.DWR_WaterID} as ${config.fieldNames.DWR_WaterID}_Lake,
+      l.${config.fieldNames.ReachCode} as ${config.fieldNames.ReachCode}_Lake,
+      st.${config.fieldNames.WaterName} as ${config.fieldNames.WaterName}_Stream,
+      st.${config.fieldNames.DWR_WaterID} as ${config.fieldNames.DWR_WaterID}_Stream,
+      st.${config.fieldNames.ReachCode} as ${config.fieldNames.ReachCode}_Stream,
+      s.${config.fieldNames.NAME} as ${STATION_NAME},
+      SPECIES = STUFF((SELECT DISTINCT ', ' + f.${config.fieldNames.SPECIES_CODE}
+                       FROM ${config.databaseSecrets.databaseName}.${config.databaseSecrets.user}.Fish_evw as f
+                       WHERE se.${config.fieldNames.EVENT_ID} = f.${config.fieldNames.EVENT_ID}
+                       FOR XML PATH ('')),
+                       1, 1, ''),
+      TYPES = STUFF((SELECT DISTINCT ', ' + eq.TYPE
+                     FROM ${config.databaseSecrets.databaseName}.${config.databaseSecrets.user}.Equipment_evw as eq
+                     WHERE se.${config.fieldNames.EVENT_ID} = eq.${config.fieldNames.EVENT_ID}
+                     FOR XML PATH ('')),
+                     1, 1, '')
+  FROM ${config.databaseSecrets.databaseName}.${config.databaseSecrets.user}.SamplingEvents_evw as se
+
+  LEFT OUTER JOIN ${config.databaseSecrets.databaseName}.${config.databaseSecrets.user}.Fish_evw as f
+  ON se.${config.fieldNames.EVENT_ID} = f.${config.fieldNames.EVENT_ID}
+
+  INNER JOIN ${config.databaseSecrets.databaseName}.${config.databaseSecrets.user}.Stations_evw as s
+  ON s.${config.fieldNames.STATION_ID} = se.${config.fieldNames.STATION_ID}
+
+  LEFT OUTER JOIN ${config.databaseSecrets.databaseName}.${config.databaseSecrets.user}.UDWRLakes_evw as l
+  ON l.${config.fieldNames.Permanent_Identifier} = s.${config.fieldNames.WATER_ID}
+
+  LEFT OUTER JOIN ${config.databaseSecrets.databaseName}.${config.databaseSecrets.user}.UDWRStreams_evw as st
+  ON st.${config.fieldNames.Permanent_Identifier} = s.${config.fieldNames.WATER_ID}
+
+  WHERE ${where}
+`;
+
+  const params = {
+    f: 'json',
+    layer: JSON.stringify({
+      id: 1,
+      source: {
+        type: 'dataLayer',
+        dataSource: {
+          type: 'queryTable',
+          workspaceId: config.dynamicWorkspaceId,
+          query: removeIrrelevantWhiteSpace(query),
+          oidFields: config.fieldNames.EVENT_ID,
+        },
+      },
+    }),
+    outFields: '*',
+    where: '1 = 1',
```

Comment:
this will remove some unnecessary %20 space encoding.
```suggestion
    where: '1=1',
```